packages/server-core/src/media/file-browser/file-browser.test.ts

```diff
@@ -216,85 +216,85 @@ describe('file-browser.test', () => {
       )
     })
 
-    it('will not create a directory in project 1 as user 2, who has only reviewer permission', async () => {
-      await app.service(projectPermissionPath).remove(null, {
-        query: {
-          userId: user2.id,
-          projectId: project1.id
-        }
-      })
-      await app.service(projectPermissionPath)._create({
-        id: v4(),
-        userId: user2.id,
-        projectId: project1.id,
-        type: 'reviewer',
-        createdAt: new Date().toISOString().slice(0, 19).replace('T', ' '),
-        updatedAt: new Date().toISOString().slice(0, 19).replace('T', ' ')
-      } as any)
-      await assert.rejects(
-        () =>
-          app.service(fileBrowserPath).create('projects/' + testProject1Name + '/public/test', {
-            user: user2
-          }),
-        {
-          message: 'Missing required project permission for ' + project1.name
-        }
-      )
-    })
-
-    it('will not create a directory in project 2 as user 2, who has only reviewer permission', async () => {
-      await app.service(projectPermissionPath).remove(null, {
-        query: {
-          userId: user2.id,
-          projectId: project2.id
-        }
-      })
-      await app.service(projectPermissionPath)._create({
-        id: v4(),
-        userId: user2.id,
-        projectId: project2.id,
-        type: 'reviewer',
-        createdAt: new Date().toISOString().slice(0, 19).replace('T', ' '),
-        updatedAt: new Date().toISOString().slice(0, 19).replace('T', ' ')
-      } as any)
-      await assert.rejects(
-        () =>
-          app.service(fileBrowserPath).create('projects/' + testProject2Name + '/public/test', {
-            user: user2
-          }),
-        {
-          message: 'Missing required project permission for ' + project2.name
-        }
-      )
-    })
-
-    it('will not create a directory in project 1 as user 2, who has only reviewer permission, if they try to use a relative path from project 2', async () => {
-      await app.service(projectPermissionPath).remove(null, {
-        query: {
-          userId: user2.id,
-          projectId: project1.id
-        }
-      })
-      await app.service(projectPermissionPath)._create({
-        id: v4(),
-        userId: user2.id,
-        projectId: project1.id,
-        type: 'reviewer',
-        createdAt: new Date().toISOString().slice(0, 19).replace('T', ' '),
-        updatedAt: new Date().toISOString().slice(0, 19).replace('T', ' ')
-      } as any)
-      await assert.rejects(
-        () =>
-          app
-            .service(fileBrowserPath)
-            .create('projects/' + testProject2Name + '/../../' + testProject1Name + '/public/test', {
-              user: user2
-            }),
-        {
-          message: 'Missing required project permission for ' + project1.name
-        }
-      )
-    })
+    // it('will not create a directory in project 1 as user 2, who has only editor permission', async () => {
```

Comment:
are these tests no longer valid?

Reply:
Nope, because now editor also has these various permissions which they previously don't. Do get in touch with Chris Ball if you have concerns giving editor access to these.

Reply:
Shouldn't we just remove all of these tests that were formerly checking reviewer access?

Reply:
I commented them out if at a later point in time we need to introduce a permission with limited permission, then maybe these tests can be leveraged.

Though I can delete if desired.